engines/scumm/he/script_v100he.cpp

```diff
@@ -747,17 +747,16 @@ void ScummEngine_v100he::o100_createSound() {
 	byte subOp = fetchScriptByte();
 
 	switch (subOp) {
-	case 0:
+	case SO_INIT:
 		_heSndResId = pop();
 		break;
-	case 53:
-		createSound(_heSndResId, -1);
+	case SO_NEW:
```

Comment:
Please add the original values as comments. E.g. `case SO_NEW: // 53`. I found it very convenient during debugging